src/azul/deployment.py

```diff
@@ -91,18 +91,20 @@ def es_endpoint(self) -> Optional[Netloc]:
             return config.es_endpoint
         else:
             self._es_config_warning('endpoint')
-            return self._describe_es_domain_status['Endpoint'], 443
+            status = self._describe_es_domain_status
+            return (status['Endpoint'], 443) if status else status
```

Comment:
```suggestion
            return None if status is None else (status['Endpoint'], 443)
```

Reply:
If we wanted to be less strict (which we don't), then your version could be abbreviated as `status and (status['Endpoint'], 443)`